packages/block-library/src/page-list-item/block.json

```
@@ -0,0 +1,50 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "core/page-list-item",
+	"title": "Page List Item",
+	"category": "design",
+	"parent": [ "core/page-list" ],
+	"description": "Display a list of all pages.",
```

I think this is inaccurate. I think it should say something like:
```suggestion
	"description": "Displays page, inside a list of all pages .",
```

---

Should it be

> Displays _a_ page...